### ISSUE TITLE
language

### ISSUE THREAD

**Issue by winniengo** [kmccallsdsu/RapTranslate#3]
I noticed in your wireframes there's a dropdown for language. Not sure exactly what you're doing with that but I'm imagining that users can select songs or translations by a given language? If so, you'll probably want to add a Language table and Songs will have a language id.

**Comment by kmccallsdsu** [comment]
Yeah, I realized this later as well and never got around to updating the schema. Will definitely add a language table and Id to each song as well as an id for users to select their default language. Good lookin' out Winnie!
